pyqtgraph/colormap.py

```diff
@@ -192,12 +194,46 @@ def getFromColorcet(name):
         color_list.append( color_tuple )
     if len(color_list) == 0:
         return None
-    cm = ColorMap(
-        pos=np.linspace(0.0, 1.0, len(color_list)),
+    cmap = ColorMap( name=name,
+        pos=np.linspace(0.0, 1.0, len(color_list)), 
         color=color_list) #, names=color_names)
-    _mapCache[name] = cm
-    return cm
-
+    _mapCache[name] = cmap
+    return cmap
+    
+def makeMonochrome(color='green'):
+    """
+    Returns a ColorMap object imitating a monochrome computer screen
+    ===============  =================================================================
+    **Arguments:**
+    color            Primary color description. Can be one of predefined identifiers
+                    'green', 'amber', 'blue', 'red', 'lavender', 'pink'
+                    or a tuple of relative R,G,B contributions in range 0.0 to 1.0
+    ===============  =================================================================
+    """
+    name = 'monochrome-'+str(color)
```

Comment:
may I suggest an f-string here?

```python
name = f"monochrome-{color}"
```

Reply:
Got point, I should get fluent with those. It does seem to work, even where the input is a tuple of floats.
--> changed.